packages/pubsub/src/v1/index.js

```diff
@@ -32,5 +33,9 @@ function v1(options) {
 }
 
 v1.SERVICE_ADDRESS = publisherClient.SERVICE_ADDRESS;
-v1.ALL_SCOPES = publisherClient.ALL_SCOPES;
+v1.ALL_SCOPES = union(
```

Comment:
This has a tendency to creep back on when clients are regenerated. So far, all scopes have been the same, so the `lodash.union` dependency isn't necessary. Can we remove?